plugin/trino-opensearch/src/main/java/io/trino/plugin/opensearch/client/IndexMetadata.java

```diff
@@ -46,7 +46,7 @@ public record PrimitiveType(String name)
     {
         public PrimitiveType
         {
-            requireNonNull(name, "name is null");
+            requireNonNull(name, "baseName is null");
```

Comment:
the variable is still called `name` here

Reply:
same on line 37